libraries/Microsoft.Bot.Connector/Authentication/CertificateAppCredentials.cs

```diff
@@ -151,11 +151,11 @@ protected override Lazy<IAuthenticator> BuildIAuthenticator()
                 LazyThreadSafetyMode.ExecutionAndPublication);
         }
 
-        private Identity.Client.IConfidentialClientApplication CreateClientApplication(X509Certificate2 clientCertificate, string appId, HttpClient customHttpClient = null)
+        private Identity.Client.IConfidentialClientApplication CreateClientApplication(X509Certificate2 clientCertificate, string appId, bool sendX5c, HttpClient customHttpClient = null)
```

Comment:
should we add default value (I assume `false`) not to make it a breaking change?

Reply:
It's just a private method used only once in this class so we are fine like this.